tests/scripts/spack_manifest/injection/inputs/prerelease.spack.yaml

```diff
@@ -4,8 +4,11 @@
 # configuration settings.
 spack:
   # add package specs to the `specs` list
+  definitions:
+  - _name: [access-om2]
+  - _version: [2024.03.0]
   specs:
```

Comment:
This has different indentation to the `spack.yaml` input. Don't know if this is relevant.